main.nf

```diff
@@ -81,12 +81,22 @@ def helpMessage() {
                                     separated by commas. Default is '21,27,33,51'
       --molecules                   Which molecule to compare on. Default is both DNA
                                     and protein, i.e. 'dna,protein,dayhoff'
-      --log2_sketch_sizes           Which log2 sketch sizes to use. Multiple are separated
-                                    by commas. Default is '10,12,14,16'
       --track_abundance             Track abundance of each hashed k-mer, could be useful for cancer RNA-seq or ATAC-seq analyses
       --skip_trimming               If provided, skip fastp trimming of reads
       --skip_compare                If provided, skip comparison of hashes using sourmash compare
 
+     Sketch size options:
+      --sketch_size                 Number of hashes to use for making the sketches.
+                                    Mutually exclusive with --sketch_size_log2
+      --sketch_size_log2            Which log2 sketch sizes to use. Multiple are separated by commas.
```

Comment:
log2 sketch size means - log2(number of hashes)

Reply:
to me personally num_hashes makes more sense then sketch_size, as size can mean so many things